.circleci/config.yml

```diff
@@ -1,52 +1,61 @@
 version: 2.1
 
-jobs:
-  build:
+executors:
+  builder:
     docker:
-      - image: rust:1.53.0
+    - image: buildpack-deps:trusty
+
+jobs:
+  docker-image:
+    executor: builder
     steps:
       - checkout
+      - setup_remote_docker
+          docker_layer_caching: true
       - run:
-          name: Version information
```

Comment:
Deleted these, already excluded in `cargo-generate.toml`

Reply:
`template.exclude` in `cargo-generate.toml` means the file does not go through the template engine but is shipped to the user directly.